commons/src/test/scala/dagr/commons/reflect/ReflectionUtilTest.scala

```diff
@@ -249,8 +384,19 @@ class ReflectUtilAnnotationTest extends UnitSpec {
     ReflectionUtil.findScalaAnnotation[ClassAnn,ClassAnnotated] shouldBe Some(new ClassAnn(c=classOf[Thread]))
   }
 
+  it should "work with Scala Enumerations" in {
+    ReflectionUtil.findScalaAnnotation[Ann,SomeEnum.type].get.getClass shouldBe classOf[Ann]
+  }
+
   "ReflectionUtil.hasScalaAnnotation" should "return if the symbol is annotated with a scala annotation" in {
     ReflectionUtil.hasScalaAnnotation[ClassAnn,ClassAnnotated] shouldBe true
     ReflectionUtil.hasScalaAnnotation[EnumAnn,ClassAnnotated] shouldBe false
   }
+
+  "ReflectionUtil.findJavaAnnotation" should "return the annotation on a class" in {
+    val value = ReflectionUtil.findJavaAnnotation(clazz=classOf[ClassJavaAnnotation], annotationClazz=classOf[TestJavaAnnotation])
+    value shouldBe 'defined
```

Comment:
Similarly, these two lines could just be `value.map(_.placeholder()) shouldBe Some("value)`